pages/your_tasks/data_organisation.md

```diff
@@ -62,6 +62,40 @@ Brief and descriptive file names are important in keeping your data files organi
   * File name: `20201203_MM_HEAD_CROPPED_V1.psd`
   * Explanation: `Time_CreatorData_TypeModification_Version`
 
+## How to choose the appropriate file formats?
+
+### Description
+File formats play an important role when you try to open files later or another person would like to work with the data. Choosing open and widely supported file formats ensures compatibility and accessibility in the foreseen future.
```

Comment:
It will also be helpful to add how it also plays a role in a context of interoperability/machine-readability.

Reply:
@ivanmicetic I extended your suggestion, please see the last commit. Please see if you are agree and feel free to modify :)